docs/versioned/install/installing-contour.md

```diff
@@ -0,0 +1,209 @@
+---
+audience: administrator
+components:
+  - serving
+function: how-to
+---
+
+# Installing Contour for Knative
+
+This page shows how to install Contour in three ways:
+
+- By using Contour’s example YAML.
+- By using the Helm chart for Contour.
+- By using the Contour gateway provisioner.
```

Comment:
We should focus on installing the Knative adapter for Contour, rather than installing Contour itself.  I'd make "Contour [installed](https://projectcontour.io/docs/1.33/deploy-options/) on the cluster" a pre-requisite, and then just talk about installing the `net-contour` controller.